features/admin.server-configurations.v1/forms/password-recovery-form/password-recovery-form.tsx

```diff
@@ -0,0 +1,709 @@
+                            className="oxygen-menu-item-chip oxygen-chip-beta" />)
+                    }
+                </Heading>
+                <Field.Checkbox
+                    ariaLabel="enableSMSBasedRecovery"
+                    name="enableSMSBasedRecovery"
+                    label={ t("extensions:manage.serverConfigurations.accountRecovery." +
+                                "passwordRecovery.form.fields.enableSMSBasedRecovery.label") }
+                    required={ false }
+                    readOnly={ readOnly }
+                    width={ 10 }
+                    disabled={ !isConnectorEnabled }
+                    listen={ (value: boolean) => setIsSMSRecoveryEnabled(value) }
+                    data-testid={ `${ testId }-sms-based-recovery` }
+                    data-componentid={ `${ testId }-sms-based-recovery` }
+                />
+                <Hint>
+                    { t("extensions:manage.serverConfigurations.accountRecovery." +
+                            "passwordRecovery.form.fields.enableSMSBasedRecovery.hint") as ReactNode }
+                </Hint>
+                <Field.Input
+                    ariaLabel="smsOtpExpiryTime"
+                    inputType="number"
+                    name="smsOtpExpiryTime"
+                    min={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .EXPIRY_TIME_MIN_VALUE
+                    }
+                    max={
+                        GovernanceConnectorConstants.PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS
+                            .SMS_OTP_EXPIRY_TIME_MAX_VALUE
+                    }
+                    label={ t("extensions:manage.serverConfigurations.accountRecovery." +
+                                "passwordRecovery.form.fields.smsOtpExpiryTime.label") }
+                    placeholder={ t("extensions:manage.serverConfigurations.accountRecovery." +
+                                        "passwordRecovery.form.fields.smsOtpExpiryTime.placeholder") }
+                    required={ false }
+                    maxLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.SMS_OTP_EXPIRY_TIME_MAX_LENGTH
+                    }
+                    minLength={
+                        GovernanceConnectorConstants
+                            .PASSWORD_RECOVERY_FORM_FIELD_CONSTRAINTS.EXPIRY_TIME_MIN_LENGTH
+                    }
+                    readOnly={ readOnly }
+                    width={ 10 }
+                    labelPosition="right"
+                    disabled={ !isSMSRecoveryEnabled }
+                    data-testid={ `${ testId }-sms-otp-expiry-time` }
+                    data-componentid={ `${ testId }-sms-otp-expiry-time` }
+                >
+                    <input/>
+                    <Label
+                        content={ "mins" }
+                    />
```

Comment:
Shall we use the native <label></label> element instead of this semantic-ui component?

We are gradually moving away from semantic-ui to oxygen-ui. Also, using native html components reduces the bundle size.

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6418/commits/9029033be45b984d5eb52fcc8af4737ae5fc3882